python/ray/experimental/channel/shared_memory_channel.py

```diff
@@ -83,6 +84,17 @@ def _create_channel_ref(
     return object_ref
 
 
+def _get_self_actor() -> Optional["ray.actor.ActorHandle"]:
+    """
+    Get the current actor handle in this worker.
+    If this is called in a driver process, it will return None.
+    """
+    try:
+        return ray.get_runtime_context().current_actor
+    except RuntimeError:
+        return None
```

Comment:
We can use `ray.get_runtime_context().get_actor_id()` instead.

Reply:
I initially considered using `ray.get_runtime_context().get_actor_id()`. However, I decided against it because it will throw a `debug` message. This is not a bug, but it may cause confusion for users.

https://sourcegraph.com/github.com/ray-project/ray@82ad4a82e3b5a20955accfdcd978de06c6de0321/-/blob/python/ray/runtime_context.py?L241

Reply:
I am not sure whether it is OK to print `debug` messages in a normal case.

Reply:
Close this comment. Feel free to reopen if my reply doesn't make sense to you.